deploy/olm-catalog/activemq-artemis-operator/0.18.0/activemq-artemis-operator.v0.18.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,974 @@
+        specDescriptors:
+          - description: Triggered by main ActiveMQArtemis CRD messageMigration entry
+            displayName: LocalOnly
+            path: localOnly
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:SingleNamespace'
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - serviceAccountName: activemq-artemis-operator
+          rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - routes
+              verbs:
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - broker.amq.io
+              resources:
+                - '*'
+              verbs:
+                - '*'
+            - apiGroups:
+                - "route.openshift.io"
+              resources:
+                - routes
+                - routes/custom-host
+                - routes/status
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+
+      deployments:
+        - name: activemq-artemis-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: activemq-artemis-operator
+            strategy:
+              rollingUpdate:
+                maxSurge: 1
+                maxUnavailable: 1
+              type: RollingUpdate
+            template:
+              metadata:
+                labels:
+                  name: activemq-artemis-operator
+              spec:
+                serviceAccountName: activemq-artemis-operator
+                containers:
+                  - name: activemq-artemis-operator
+                    image: quay.io/artemiscloud/activemq-artemis-operator:0.18.0
+                    args:
+                      # Explicitly set the logging level.
+                      # Valid values are debug, info, and error
+                      # from most to least.
+                      # If running entrypoint_debug then use '-- --zap-level debug'
+                      - '--zap-level debug'
```

Comment:
@gaohoward We should probably set this to info.

Reply:
+1, will change that